### PR TITLE
Feature/ui update

### PR DESCRIPTION
**Pull request by tmart2322**
Added sidemenu bar for users who use mobile

Edited DJ Event Page to switch between Side Menu and regular Navbar based on screen width, made DJ Event Page lists resize based on size of screen, and adjusted Home Page Navbar to look more uniform with the other pages

### CODE REVIEW

File: jukebox_dj/index/static/jukebox-dj/src/app/app.scss [hunk 1]
```diff
@@ -47,3 +47,14 @@
     text-align: center;
   }
 }
+
+.mobile-toolbar .toolbar-background {
```
Comment (jakeharding) [inline jukebox_dj/index/static/jukebox-dj/src/app/app.scss]: This toolbar-background class definition is repeated in a few places. The definition in app.scss should make it effective for the whole app then any overrides can be done in child pages. Do we need the other definitions?

File: jukebox_dj/index/static/jukebox-dj/src/pages/dj-event/dj-event.scss [hunk 2]
```diff
@@ -1,3 +1,22 @@
 page-dj-event {
+  @media (max-width: 575px) {
+    .hidden-xs-down {
+      display: none !important;
+    }
+  }
+  @media (min-width: 576px) {
```
Comment (jakeharding) [inline jukebox_dj/index/static/jukebox-dj/src/pages/dj-event/dj-event.scss]: These line up with the Ionic sass small screen breakpoints. Is that where you got them?

File: jukebox_dj/index/static/jukebox-dj/src/app/app.scss (hunk 1 above)
Comment (tmart2322) [reply jukebox_dj/index/static/jukebox-dj/src/app/app.scss]: I wasn't sure if we wanted to change the background color for every page but if we want to I can definitely do that.

File: jukebox_dj/index/static/jukebox-dj/src/pages/dj-event/dj-event.scss (hunk 2 above)
Comment (tmart2322) [reply jukebox_dj/index/static/jukebox-dj/src/pages/dj-event/dj-event.scss]: Yes I got them from there.

Comment (jakeharding) [reply jukebox_dj/index/static/jukebox-dj/src/pages/dj-event/dj-event.scss]: Cool. Minor detail but let's use map-get scss function to get the breakpoints.

**Pull request merged**